tacitDB_Mock.go

```diff
@@ -17,6 +17,7 @@ type tacitDBMock struct {
 	//Behavioral Setup
 	firstResultDBUser *dbUser
 	hasError          bool
+	noRecordFound     bool
```

Comment:
Seems kind of nit picky but I am wondering what you think about putting this in the affirmative like `recordFound` to make future test setup a bit more straight forward?